test/fences-builder.js

```diff
@@ -56,6 +56,20 @@ describe('fences-builder', function () {
       return array.sort(function (a, b) {
         if (a.name < b.name) { return -1; }
         if (a.name > b.name) { return 1; }
+
+        function sum(previousValue, currentValue) {
+          return previousValue + currentValue;
+        }
+        var avgA = a.geometry.coordinates.reduce(sum);
+        var avgB = b.geometry.coordinates.reduce(sum);
```

Comment:
oops, i started out wanting to make an avg, but then decided sum would be good enough. i'll update the variable names. good catch.